plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -545,14 +546,16 @@ public void updateTableStatistics(HiveIdentity identity, String databaseName, St
                     return Stream.of(createMetastoreColumnStatistics(entry.getKey(), type, entry.getValue(), rowCount));
                 })
                 .collect(toImmutableList());
+        // If the table is a transactional table, the column statistics will be empty for all
```

Comment:
It this still valid if `currentStatistics` above were non-empty initially?

Reply:
Good question.  Do you think it matters to the outcome?  Maybe it's just the comment that is wrong in that case.

Reply:
The logic seems rational. Yeah - I think it is just about comment being not accurate (as well as commit message which claims logic is only used only for ANALYZE).

Reply:
Can we have a test which first performs ANALYZE on a table and then checks if statistics are still there after a INSERT/UPDATE/DELETE is made on the table? Or maybe there is one already?